src/views/inspector/InspectorViewProvider.ts

```diff
@@ -7,7 +7,7 @@ import { INSPECTOR_VIEW_BUTTONS } from '../../components/hoverCard'
 
 type InspectorViewMessage =
   | { type: 'variableOrFeature', value: 'Variable' | 'Feature' }
-  | { type: 'key', value: string, buttonType?: INSPECTOR_VIEW_BUTTONS }
+  | { type: 'key', value: string, buttonType?: INSPECTOR_VIEW_BUTTONS, selectedType?: 'Variable' | 'Feature' }
```

Comment:
will this ever need to be used for Features?

Reply:
could be, but its not right now. Ill remove that option